assets/js/components/wp-dashboard/WPDashboardWidgets.js

```diff
@@ -85,110 +75,42 @@ const WPDashboardWidgets = () => {
 	const analyticsModuleActiveAndConnected =
 		analyticsModuleActive && analyticsModuleConnected;
 
-	// The two Analytics widgets at the top can be combined (i.e. the second can be hidden)
-	// if they are both ReportZero.
-	const shouldCombineAnalyticsArea1 = useSelect(
-		( select ) =>
-			select( CORE_WIDGETS ).getWidgetState( WIDGET_VISITORS )
-				?.Component === ReportZero &&
-			select( CORE_WIDGETS ).getWidgetState( WIDGET_SESSION_DURATION )
-				?.Component === ReportZero
-	);
-
-	// The Analytics widget at the bottom can be combined / hidden if one of the two at the top
-	// is also ReportZero.
-	const shouldCombineAnalyticsArea2 = useSelect(
-		( select ) =>
-			( select( CORE_WIDGETS ).getWidgetState( WIDGET_VISITORS )
-				?.Component === ReportZero &&
-				select( CORE_WIDGETS ).getWidgetState( WIDGET_POPULAR_PAGES )
-					?.Component === ReportZero ) ||
-			( select( CORE_WIDGETS ).getWidgetState( WIDGET_SESSION_DURATION )
-				?.Component === ReportZero &&
-				select( CORE_WIDGETS ).getWidgetState( WIDGET_POPULAR_PAGES )
-					?.Component === ReportZero )
-	);
-
-	// The Search Console widgets can be combined (i.e. the second is hidden) if they are both
-	// ReportZero.
-	const shouldCombineSearchConsoleWidgets = useSelect(
-		( select ) =>
-			select( CORE_WIDGETS ).getWidgetState( WIDGET_IMPRESSIONS )
-				?.Component === ReportZero &&
-			select( CORE_WIDGETS ).getWidgetState( WIDGET_CLICKS )
-				?.Component === ReportZero
-	);
-
-	const zeroDataStates = useFeature( 'zeroDataStates' );
-
-	if (
-		analyticsModule === undefined ||
-		shouldCombineAnalyticsArea1 === undefined ||
-		shouldCombineAnalyticsArea2 === undefined ||
-		shouldCombineSearchConsoleWidgets === undefined
-	) {
+	if ( analyticsModule === undefined ) {
 		return null;
 	}
 
 	return (
 		<div
-			className={ classnames( 'googlesitekit-wp-dashboard-stats', {
-				'googlesitekit-wp-dashboard-stats--fourup':
-					analyticsModuleActive && analyticsModuleConnected,
-				'googlesitekit-wp-dashboard-stats--twoup': zeroDataStates,
-			} ) }
+			className={ classnames(
+				'googlesitekit-wp-dashboard-stats googlesitekit-wp-dashboard-stats--twoup',
+				{
+					'googlesitekit-wp-dashboard-stats--fourup':
+						analyticsModuleActive && analyticsModuleConnected,
+				}
+			) }
 		>
 			<WPDashboardIdeaHub />
 
 			{ analyticsModuleActiveAndConnected && (
 				<Fragment>
 					<WPDashboardUniqueVisitorsWidget />
-					{ ! shouldCombineAnalyticsArea1 && (
-						<WPDashboardSessionDurationWidget />
-					) }
+					<WPDashboardSessionDurationWidget />
 				</Fragment>
 			) }
 
 			<Fragment>
```

Comment:
I am not sure why it's here - maybe it was historically a grouping for a conditional render - but this `Fragment` is not needed and can be removed, obviously retaining the children.